games-board/cockatrice/metadata.xml

```diff
@@ -16,6 +16,8 @@
 	</upstream>
 	<use>
 		<flag name="server">Compile server support</flag>
+		<flag name="oracle">Build with oracle</flag>
```

Comment:
Is this about Oracle database, or something completely different? If the former, you can remove the desc since it's a global flag. If the latter, you should be more specific.